internal/infrastructure/bot/commands.go

```diff
@@ -128,6 +134,20 @@ func (b *Bot) getCommands() []*discordgo.ApplicationCommand {
 			Description: "Request a summary snapshot",
 			Type:        discordgo.ChatApplicationCommand,
 		},
+		{
+			Name:        "spot-summary",
+			Description: "Request a summary snapshot for a specific respawn (DM)",
+			Type:        discordgo.ChatApplicationCommand,
+			Options: []*discordgo.ApplicationCommandOption{
+				{
+					Name:         "respawn",
+					Description:  "Name of the respawn",
+					Type:         discordgo.ApplicationCommandOptionString,
+					Required:     true,
+					Autocomplete: true,
+				},
+			},
+		},
```

Comment:
I have a feeling adding a new command for the same request is a bit redundant. Could we instead just add an optional field, `spot`, to the already existing `/summary` command, so that if it's filled, it'll filter the spot for the user, and if it's not - it will provide the whole summary?